omniauth-shopify-oauth2.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'omniauth-oauth2', '~> 1.2'
-  s.add_runtime_dependency 'oauth2', '~> 1.1.0'
+  s.add_runtime_dependency 'omniauth-oauth2', '~> 1.4.0'
+  s.add_runtime_dependency 'oauth2', '~> 1.4.0'
```

Comment:
Let's remove this dependency, we can let `omniauth-oauth2` resolve it. It was added only because v1.2.0 of `oauth2` introduced an issue on encoding which was later fixed by https://github.com/intridea/oauth2/pull/271